### PR TITLE
Rhel9 stig prep

### PR DESCRIPTION
**Pull request by Mab879**
#### Description:

Various changes for the RHEL9 STIG. 

See each commit for details.

#### Rationale:

RHEL9 STIG

### CODE REVIEW

File: linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_faillock_deny_root/rule.yml [hunk 1]
```diff
@@ -103,3 +103,6 @@ warnings:
         be shown in the remediation report.
         If the system supports the <tt>/etc/security/faillock.conf</tt> file, the pam_faillock
         parameters should be defined in <tt>faillock.conf</tt> file.
+
+srg_requirement: |-
+    {{{ full_name }}} must automatically lock the root account until the root account is released by an administrator when three unsuccessful logon attempts occur during a 15-minute time period.
```
Comment (jan-cerny) [inline linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_faillock_deny_root/rule.yml]: No newline at the end of file

File: linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_faillock_interval/rule.yml [hunk 2]
```diff
@@ -84,6 +84,9 @@ ocil: |-
     The output should show <tt>fail_interval = &lt;interval-in-seconds&gt;</tt> where <tt>interval-in-seconds</tt> is <tt>{{{ xccdf_value("var_accounts_passwords_pam_faillock_fail_interval") }}}</tt> or greater.
     {{% endif %}}
 
+srg_requirement: |-
+    {{{ full_name }}} must automatically lock an account when three unsuccessful logon attempts occur within {{{ xccdf_value("var_accounts_passwords_pam_faillock_fail_interval") }}} secconds.
```
Comment (jan-cerny) [inline linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_faillock_interval/rule.yml:88]: secconds

File: linux_os/guide/system/auditing/auditd_configure_rules/file_permissions_var_log_audit/rule.yml [hunk 3]
```diff
@@ -76,3 +76,5 @@ fixtext: |-
     $ sudo chmod 0600 $log_file
     $ sudo chmod 0400 $log_file.*
 
+srg_requirement: |-
+   {{{ full_name }}} audit log file must have mode 640 less permissive to prevent unauthorized access to the audit log.
```
Comment (jan-cerny) [inline linux_os/guide/system/auditing/auditd_configure_rules/file_permissions_var_log_audit/rule.yml]: newline end of file

Comment (jan-cerny) [inline linux_os/guide/system/auditing/auditd_configure_rules/file_permissions_var_log_audit/rule.yml]: or less permissive

File: linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_minimum_age_login_defs/rule.yml [hunk 4]
```diff
@@ -70,5 +70,8 @@ fixtext: |-
 
     PASS_MIN_DAYS {{{ xccdf_value("var_accounts_minimum_age_login_defs") }}}
 
+srg_requirement: |-
+    {{{ full_name }}} passwords for new users or password changes must have a 24 hours/1 day minimum password lifetime restriction in /etc/logins.def.
```
Comment (jan-cerny) [inline linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_minimum_age_login_defs/rule.yml:74]: login.defs

File: linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_maximum_age_login_defs/rule.yml [hunk 5]
```diff
@@ -79,6 +79,8 @@ fixtext: |-
     Add, or modify the following line in the "/etc/login.defs" file:
 
     PASS_MAX_DAYS {{{ xccdf_value("var_accounts_maximum_age_login_defs") }}}
+srg_requirement: |-
+    {{{ full_name }}} user account passwords for new users or password changes must have a 60 day maximum password lifetime restriction in /etc/logins.def.
```
Comment (jan-cerny) [inline linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_maximum_age_login_defs/rule.yml:77]: login.defs

**Review by jan-cerny — CHANGES_REQUESTED** [review]

File: linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_faillock_deny_root/rule.yml (hunk 1 above)
Comment (ggbecker) [reply linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_faillock_deny_root/rule.yml]: I believe it github explicitly mentions when there is no newline at the end o the file, which is not the case here.

Comment (jan-cerny) [reply linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_faillock_deny_root/rule.yml]: I went commit by commit and the red sign is shown there, but when I see the whole diff it isn't shown there, probably it was fixed by a later commit, so if there is a newline at the end of file then please disregard it.

**Comment by ggbecker** [comment]
@jan-cerny I have address the feedback on Matthew's behalf and did some cleanup as you can see in [0820cca](https://github.com/ComplianceAsCode/content/pull/8708/commits/0820cca71da750afcbc6d280d658eb5845b9ada5)

**Review by jan-cerny — APPROVED** [review]

**Pull request merged**